### DISCUSSION TITLE
libcurl error executing build.cmd

### DISCUSSION THREAD

**Discussion by EatingEdu** [microsoft/AirSim#4216] [Support Q&A] [answered]
## Bug report 

- AirSim Version/#commit:
- UE/Unity version: 4.25.4
- autopilot version:
- OS Version:win10

### What's the issue you encountered?

according to the tips:https://microsoft.github.io/AirSim/unreal_blocks/#windows
when i download the Airsim , I run the build.cmd (done); i run update_from_git.bat(done);when i enter step 4 in the url, i open the env  in UE and choose  the quartor instead of car . a line jump : 

Error at startup: bind: ????????????????????????????????????????????????????: ????????????????????????????????????????????????????
and python can't connect with the airsim env ;
i need some help about this question.

**Comment by jonyMarino** [comment]
Hi and welcome @EatingEdu! Can you open the visual studio solution for the Blocks project and press the run button to get a more descriptive error?

**Comment by EatingEdu** [comment]
![image](https://user-images.githubusercontent.com/72775651/141071738-9eef8c81-01ce-48ff-af49-b56a6d3b1c66.png)

**Comment by jonyMarino** [comment, marked as answer]
@EatingEdu this is an unreal issue. Try with [this](https://forums.unrealengine.com/t/what-is-my-editor-trying-to-do/76707/22)

**Comment by EatingEdu** [comment]
> [](https://forums.unrealengine.com/t/what-is-my-editor-trying-to-do/76707/22)

ok,thank you ! i restart the pc ,and it worked. i have done according to the link you have recommanded. When i find this question again , i will reply this. Thank you for your reply again.
